gnucash/import-export/aqb/assistant-ab-initial.c

```diff
@@ -290,6 +305,49 @@ aai_wizard_button_clicked_cb(GtkButton *button, gpointer user_data)
     LEAVE(" ");
 }
 
+#ifdef AQBANKING6
+guint
+aai_ab_account_hash (gconstpointer v)
+{
+	if (v == NULL)
+		return 0;
+	else
+		/* Use the account unique id as hash value */
+		return AB_AccountSpec_GetUniqueId((const GNC_AB_ACCOUNT_SPEC *) v);
+}
+
+gboolean
+aai_ab_account_equal (gconstpointer v1, gconstpointer v2)
+{
+	if (v1 == NULL || v2 == NULL)
+		return v1 == v2;
+	else
+	{
+		/* Use the account unique id to check for equality */
+		uint32_t uid1 = AB_AccountSpec_GetUniqueId((const GNC_AB_ACCOUNT_SPEC *) v1);
+		uint32_t uid2 = AB_AccountSpec_GetUniqueId((const GNC_AB_ACCOUNT_SPEC *) v2);
+		return uid1 == uid2;
+	}
+}
+#endif
+
+static void
+insert_acc_cb(gpointer key, gpointer value, gpointer user_data)
+{
+       GNC_AB_ACCOUNT_SPEC *ab_acc = (GNC_AB_ACCOUNT_SPEC *) key;
+       GHashTable *accounts = (GHashTable *) user_data;
+       g_return_if_fail(accounts && value && ab_acc);
+       g_hash_table_insert(accounts, value, (gpointer) ab_acc);
```

Comment:
OK, I will remove it.

Reply:
Since as you say below that you only need revhash's key then store it in both key and value, g_hash_table_insert recognizes and optimizes for that special case.